User/server/route/user.js

```diff
@@ -1,8 +1,8 @@
 const express = require('express');
 const router = express.Router();
 
-const userController = require('../../../backend/server/controllers/userController.js');
-const middleware = require('../middleware.js');
+const userController = require('../../../User/server/controller/userController.js');
+const middleware = require('../../../User/server/middleware.js');
```

Comment:
I propose using absolute imports (along with with relative imports). This help help to reduce the `../` that we see.

Reading on [JS import hell](https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e) (Relative vs absolute imports).

I have already used something like this in my [UPM](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g32/blob/assignment-4-lhy/upm/jsconfig.json).

Reply:
Other parts of the code can also benefit from using absolute imports, to name a few `userService.js` `userRepoMySql.js` `AuthApi.js`

Reply:
Resolved in #149